### PR TITLE
Add WFA quicktrack control application

### PR DESCRIPTION
**Pull request by krish2718**
This application is useful to our customers going for their products certification using our qualified solutions.

### CODE REVIEW

**Review by sachinthegreen — APPROVED** [review]

**Review by rlubos — COMMENTED** [review]
What about the sample documentation? @richabp

**Comment by krish2718** [comment]
> What about the sample documentation? @richabp

Missed that, pushing now, thanks.

File: samples/wifi/wfa_qt_app/README.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
+.. _wifi_wfa_qt_app_sample:
+
+Wi-Fi: WFA Quicktrack control app
```
Comment (kittydepa) [inline samples/wifi/wfa_qt_app/README.rst:3]:
```suggestion
Wi-Fi: WFA QuickTrack control application
```

File: samples/wifi/wfa_qt_app/sample.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+sample:
+  description: WFA Quicktrack control application
```
Comment (kittydepa) [inline samples/wifi/wfa_qt_app/sample.yaml:2]:
```suggestion
  description: WFA QuickTrack control application
```

File: samples/wifi/wfa_qt_app/README.rst [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+#################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates how to use the WFA Quicktrack (WFA QT) library needed for Wi-Fi Alliance Quicktrack certification.
```
Comment (kittydepa) [inline samples/wifi/wfa_qt_app/README.rst:10]:
```suggestion
This sample demonstrates how to use the WFA QuickTrack (WFA QT) library needed for Wi-Fi Alliance® QuickTrack certification.
```

File: samples/wifi/wfa_qt_app/README.rst [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+This sample acts as a control application for the WFA QT library, to be used for Wi-Fi Alliance Quicktrack certification. See `Wi-Fi Alliance Certification for nRF70 Series <Wi-Fi Alliance Certification for nRF70 Series_>`_
```
Comment (kittydepa) [inline samples/wifi/wfa_qt_app/README.rst:22]:
```suggestion
This sample acts as a control application for the WFA QT library, to be used for Wi-Fi Alliance QuickTrack certification.
See `Wi-Fi Alliance Certification for nRF70 Series <Wi-Fi Alliance Certification for nRF70 Series_>`_
```

**Review by kittydepa — CHANGES_REQUESTED** [review]

**Review by kittydepa — APPROVED** [review]

**Pull request merged**